doc/computation.rst

```diff
@@ -225,6 +225,23 @@ If the result would be empty, an error is raised instead:
     In [1]: arr[:2] + arr[2:]
     ValueError: no overlapping labels for some dimensions: ['x']
 
+However, one can explicitly change this default automatic alignment type ("inner")
+via :py:func:`~xarray.set_options()`
+
+.. ipython:: python
+
+    xr.set_options(arithmetic_join="outer")
```

Comment:
Can you only show the example using `with`?

I'm now wondering if we need add some mechanism to *require* using the context manager for `arithmetic_join`. If users don't use a context manager, very mysterious/nonlocal bugs could appear when this changes the behavior of unrelated code (e.g., in an import statement).